tests/network-setups/.gitignore

```diff
@@ -4,6 +4,8 @@
 fabric/dev/config
 fabric/dev/bin
 fabric/dev/.fabric-setup
+fabric/dev/install-fabric.sh
```

Comment:
Why have this?

Reply:
the new way of downloading fabric binaries and images, is downloading this script and this script then downloads rest of stuff.
Hence added the script in gitignore, because this script will be downloaded everytime the testnet is started.